src/config-schema.coffee

```diff
@@ -112,7 +112,16 @@ module.exports =
         description: 'Allow items to be previewed without adding them to a pane permanently, such as when single clicking files in the tree view.'
         type: 'boolean'
         default: true
-
+      telemetryConsent:
+        description: 'Allow usage statistics and exception reports to be sent to the Atom team to help improve the product.'
+        title: 'Send Telemetry to the Atom Team'
+        type: 'string'
+        default: 'undecided'
+        enum: [
+          {value: 'limited', description: 'Allow limited anonymous usage stats, exception and crash reporting'}
```

Comment:
@damieng why is there no option like: `unlimited` ? I want to make Atom better and I mean business.